src/style.css

```diff
@@ -46,13 +42,25 @@ body {
   justify-content: space-around;
 }
 
-.nav__link.is-active {
+.nav__link {
+  font-weight: 500;
+  min-height: 60px;
+  align-items: center;
+  display: flex;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: black;
+  font-size: 12px;
+  margin-left: 20px;
+}
+
+.is-active {
   color: var(--color__nav);
   position: relative;
```

Comment:
You are altering `position` for active links

In your case it is a bad idea - I would suggest making the same `position` for all `nav_link`s as changing
* gives browser additional unnecessary job
* may break your layout one day
* may look confusing for other developers